addons/full-upgrade/run-upgrade.sh

```diff
@@ -76,6 +101,7 @@ function apt_upgrade_packetfence_package() {
   else
     DEBIAN_FRONTEND=noninteractive apt install -q -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" packetfence -y
   fi
+  DEBIAN_FRONTEND=noninteractive apt autoremove -q -y
```

Comment:
=> TO BE MOVED in the `elif is_deb_based; then`

Maybe we should add a flag on this one.
To execute it only if everything is fine for exemple.

Reply:
IMHO, this is not necessary because the command itself already behave like this:
* useless packages to remove: they are removed
* no useless packages to remove: command do nothing

Reply:
Ok then, it is fine with me.